magefiles/magefile.go

```diff
@@ -181,6 +182,10 @@ func (Test) FixtureVMImages() error {
 	return fixtureVMImages()
 }
 
+func (Test) FixtureTerraformPlanSnapshots() error {
```

Comment:
IIUC this function is needed to re-generate tfplan files.
Can you add a comment on this?

This will make this code easier to maintain

Reply:
@simar7 I have AWS set to localstack by default. Fixed it https://github.com/aquasecurity/trivy/pull/6176/commits/5722384658fb0ff995ad4fb68267aa0a0651da26